src/components/data/data-utils.js

```diff
@@ -369,13 +374,13 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
           onClick: doUpload
         }, ['Start Import Job'])
       }, [
-        div({ style: { padding: '0 0 1rem' } },
+        div(
           ['Choose the data import option below. ',
             h(Link, {
               ...Utils.newTabLinkProps,
               href: 'https://support.terra.bio/hc/en-us/articles/360025758392'
             }, ['Click here for more info on the table.']),
-            p(['Data will be saved in location: 🇺🇸 ', span({ style: { fontWeight: 'bold' } }, 'US '), '(Terra-managed).'])]),
+            p(['Data will be saved in location: 🇺🇸  ', span({ style: { fontWeight: 'bold' } }, regionLabelToDisplay), ' (Terra-managed).'])]),
```

Comment:
super minor nit: is the added space after the flag intentional? I realize the one before `(Terra-managed)` is necessary.

Reply:
This was intentional -- just the spacing was quite close between all the values in the UI

Reply:
Do we want to hard-code the flag as well?  I see there's a `getRegionFlag` method alongside the `getRegionLabel` method in `azure-utils.js`.

Reply:
I held off handling the flag value since it is not Azure-specific (e.g. applies to a Google workspace as well)